lidar_watchdog:
  ros__parameters:
    input_topic: "scan"
    threshold_ms: 500
    error_code: 10
    enable_debug_msg: true

velocity_watchdog:
  ros__parameters:
    input_topic: "cmd_vel"
    threshold_ms: 500
    error_code: 20
    enable_debug_msg: false